src/screens/FeedScreen/RenderList.js

```diff
@@ -250,7 +250,7 @@ const RenderListFeed = (props) => {
                 time={item.latest_reactions.comment[0].created_at}
                 totalComment={getTotalReaction(item) - 1}
                 item={item.latest_reactions.comment[0]}
-                onPress={onPress}
+                onPress={() => onPressComment(isHaveSeeMore)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be passing a new parameter `isHaveSeeMore` to the `onPressComment` function. However, without knowing what this parameter does or how it affects the function, it's hard to say if this is a good change or not. Please ensure that the `onPressComment` function can handle this new parameter and that it doesn't introduce any unexpected behavior.
```diff
- onPress={onPress}
+ onPress={() => onPressComment(isHaveSeeMore)}
```
If `onPressComment` function is not expecting `isHaveSeeMore` as a parameter, you might need to modify the function definition accordingly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->